src/index.ts

```diff
@@ -20,8 +24,14 @@ const run = async () => {
     jsonSummary, jsonFinal 
   });
 
+  // Indicate working directory (if set) in Action summary for easier understanding (ie. in a monorepo)
+  let summaryHeading = "Coverage Summary";
+  if (workingDirectory) {
+    summaryHeading += ` for \`${workingDirectory}\``;
+  }
+
   const summary = core.summary
-    .addHeading('Coverage Summary')
+    .addHeading(summaryHeading, 2)
```

Comment:
Using an `h1` tag seemed a bit excessive (typically should be only one per page) 🤷

Reply:
True, good catch!